### PR TITLE
#122-implemented books' details page

### LINKED ISSUE #122: Books' Details Page Implementation

### PR DESCRIPTION
**Pull request by Aarrtteemm123**
#122 - implemented books' details page (based on books' details page from the administration site)

### CODE REVIEW

File: library/catalog/templates/book-details.html [hunk 1]
```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8"/>
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css">
+    <title>Book-details</title>
+    <style>
```
Comment (ivan-dubei) [inline library/catalog/templates/book-details.html]: From my point of view, it is a good practice to have a CSS part of the code in the seperate file

File: library/catalog/views.py [hunk 2]
```diff
@@ -1,17 +1,24 @@
 from django.http import HttpResponse
 from django.shortcuts import redirect, render
 
-from catalog.forms import RegistrationForm
+from admin.models import Book
```
Comment (ivan-dubei) [inline library/catalog/views.py]: Are you totaly sure that we can use models from the Flask part ?

File: library/library/settings.py [hunk 3]
```diff
@@ -121,10 +121,10 @@
 # https://docs.djangoproject.com/en/3.2/howto/static-files/
 
 STATIC_URL = '/static/'
-STATICFILES_DIRS = [
-    BASE_DIR / 'static',
-    '/var/www/static/',
-]
+#STATICFILES_DIRS = [
```
Comment (ivan-dubei) [inline library/library/settings.py]: Skipping a chunk of the code with the help of coments is a bad practice.

File: library/catalog/templates/book-details.html (hunk 1 above)
Comment (Aarrtteemm123) [reply library/catalog/templates/book-details.html]: Yes, but it has different behavior in that case

**Review by taras0024 — APPROVED** [review]

**Pull request merged**